Resources/Prototypes/Entities/Mobs/Species/skeleton.yml

```
@@ -102,6 +102,8 @@
           probability: 0.5
   - type: FireVisuals
     alternateState: Standing
+  - type: Tag
```

yamllint

[error] wrong indentation: expected 4 but found 2

(indentation)